packages/covector/__snapshots__/index.test.ts.snap

```diff
@@ -1514,6 +1514,94 @@ Object {
 }
 `;
 
+exports[`integration test for preview command runs version and publish for js and rust 1`] = `
+Object {
+  "consoleInfo": Array [],
+  "consoleLog": Array [
+    Array [
+      "bumping package-b to 0.0.1-branch-name.12345 to publish a preview",
+    ],
+    Array [
+      "bumping package-a to 0.0.1-branch-name.12345 to publish a preview",
+    ],
+    Array [
+      "package-a [publish]: echo publishing package-a would happen here",
+    ],
+    Array [
+      "publishing package-a would happen here",
+    ],
+    Array [
+      "package-b [publish]: echo publishing package-b would happen here",
+    ],
+    Array [
+      "publishing package-b would happen here",
+    ],
```

Comment:
In this new fixture I have `package-a` and `package-b` and `package-a` has `package-b` as one of its dependencies. I added a changefile only for `package-b` to see if it'll bump the dependency _inside_ `package-a`.

Reply:
So maybe the issue we saw earlier comes from when different managers are mixed in together?

Reply:
> So maybe the issue we saw earlier comes from when different managers are mixed in together?

That would be surprising to me. It might be worth adding a `package-c` with the same manager as your change to confirm?

Reply:
should package-c have any dependencies? or have it be added as a dependency to package-a or package-b?